benchmark/experimenting/utils.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import os
+import stat
+
+from pathlib import Path
+from shutil import copyfile
+
+
+def copy_binary(binary: Path, target: str):
+    logging.info(f'Copying binary from {binary}...')
```

Comment:
`f'Copying binary from {binary} to {target}...'`

Reply:
done